sentry_sdk/serializer.py

```diff
@@ -123,6 +124,17 @@ def serialize(event, **kwargs):
     )  # type: bool
     max_value_length = kwargs.pop("max_value_length", None)  # type: Optional[int]
     is_vars = kwargs.pop("is_vars", False)
+    custom_repr = kwargs.pop("custom_repr", None)  # type: Callable[..., Optional[str]]
+
+    def _safe_repr_wrapper(value):
+        # type: (Any) -> str
+        try:
+            repr_value = None
+            if custom_repr is not None:
+                repr_value = custom_repr(value)
+            return repr_value or safe_repr(value)
```

Comment:
Empty strings should probably still count as valid repr:
```suggestion
            return repr_value if repr_value is not None else safe_repr(value)
```